python/lsst/ip/isr/assembleCcdTask.py

```diff
@@ -195,13 +197,15 @@ def assembleCcd(self, assembleInput):
         ccd = None
         if hasattr(assembleInput, "has_key"):
             # Get a detector object for this set of amps
-            ccd = assembleInput.itervalues().next().getDetector()
+            ccd = assembleInput.values().next().getDetector()
```

Comment:
Ok, after looking at it a bit more, the object should be a dictionary in that case, so __next__ should be available. I am rather suspicious of this whole function, but I'm not going to root around in it.